<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JIRA Explorer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icon.ico</normaloff>:/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayoutCentral">
    <item>
     <widget class="QSplitter" name="splitterMain">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTreeView" name="treeTickets">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollDetails">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollDetailsWidget">
        <layout class="QVBoxLayout" name="verticalLayoutDetails">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutHeader">
           <item>
            <widget class="QLabel" name="labelSelectedKey">
             <property name="text">
              <string>(no ticket selected)</string>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred"/>
             </property>
             <property name="font">
              <font>
               <pointsize>18</pointsize>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelSelectedStatus">
             <property name="text">
              <string></string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonOpenInJira">
             <property name="text">
              <string>Open in Jira</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="labelSelectedSummary">
           <property name="text">
            <string></string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupTransition">
           <property name="title">
            <string>Transition</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayoutTransition">
            <item>
             <widget class="QLabel" name="labelTransitionTo">
              <property name="text">
               <string>To:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboTransitions"/>
            </item>
            <item>
             <widget class="QPushButton" name="buttonApplyTransition">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitterDescDetails">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="childrenCollapsible">
            <bool>false</bool>
           </property>
           <widget class="QGroupBox" name="groupDescription">
            <property name="title">
             <string>Description</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayoutDescription">
             <item>
              <widget class="QTextEdit" name="textDescription"/>
             </item>
             <item>
              <widget class="QPushButton" name="buttonSaveDescription">
               <property name="text">
                <string>Save Description</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="groupDetails">
            <property name="title">
             <string>Details</string>
            </property>
            <layout class="QFormLayout" name="formLayoutDetails">
             <item row="0" column="0">
              <widget class="QLabel" name="labelStoryPoints">
               <property name="text">
                <string>Story Points</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QWidget" name="widgetStoryPointsRow">
               <layout class="QHBoxLayout" name="horizontalLayoutStoryPoints">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="lineStoryPoints"/>
                </item>
                <item>
                 <widget class="QPushButton" name="buttonUpdateStoryPoints">
                  <property name="text">
                   <string>Update</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelAssignee">
               <property name="text">
                <string>Assignee</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QWidget" name="widgetAssigneeRow">
               <layout class="QHBoxLayout" name="horizontalLayoutAssignee">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="lineAssignee"/>
                </item>
                <item>
                 <widget class="QPushButton" name="buttonUpdateAssignee">
                  <property name="text">
                   <string>Update</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="labelSprintId">
               <property name="text">
                <string>Sprint ID</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QWidget" name="widgetSprintRow">
               <layout class="QVBoxLayout" name="verticalLayoutSprint">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="lineSprintId"/>
                </item>
                <item>
                 <widget class="QLabel" name="labelCurrentSprintName">
                  <property name="text">
                   <string></string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="buttonUpdateSprint">
                  <property name="text">
                   <string>Update</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="labelDueDate">
               <property name="text">
                <string>Due Date</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QWidget" name="widgetDueDateRow">
               <layout class="QHBoxLayout" name="horizontalLayoutDueDate">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QDateEdit" name="dateDueDate">
                  <property name="calendarPopup">
                   <bool>true</bool>
                  </property>
                  <property name="displayFormat">
                   <string>yyyy-MM-dd</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="buttonUpdateDueDate">
                  <property name="text">
                   <string>Update</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupAddComment">
           <property name="title">
            <string>Add Comment</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayoutAddComment">
            <item>
             <widget class="QTextEdit" name="textNewComment">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>120</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonPostComment">
              <property name="text">
               <string>Post Comment</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupComments">
           <property name="title">
            <string>Comments</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayoutComments">
            <item>
             <widget class="QListWidget" name="listComments"/>
            </item>
            <item>
             <widget class="QPushButton" name="buttonEditComment">
              <property name="text">
               <string>Edit Selected Comment...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupHistory">
           <property name="title">
            <string>Activity History</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayoutHistory">
            <item>
             <widget class="QListWidget" name="listHistory"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
